src/model/dashboard.cpp

```diff
@@ -21,7 +17,7 @@ std::vector<record_dashboard_t> record_dashboard_t::getDashboards() {
 }
 
 record_dashboard_t record_dashboard_t::updateDashboard(record_dashboard_t recordToUpdate) {
-    SM::getDatabase()->updateDashboard(std::move(recordToUpdate));
```

Comment:
removed std::move because we are using the return value, this strips the std::string name field, did the same for updateDevice()